pkg/sensors/tracing/generickprobe.go

```diff
@@ -466,7 +466,7 @@ func preValidateKprobe(
 		}
 	}
 
-	if selectors.HasSigkillAction(f) && !config.EnableLargeProgs() {
+	if selectors.HasAction(f, "sigkill") && !config.EnableLargeProgs() {
```

Comment:
maybe using `ActionType...` constants would be less error prone, like:
`
if selectors.HasAction(f, ActionTypeSigKill) 
`